app/code/Magento/MediaGalleryUi/Test/Mftf/Test/AdminMediaGalleryUploadCategoryImageTest.xml

```diff
@@ -37,9 +37,8 @@
         <actionGroup ref="AdminSaveCategoryFormActionGroup" stepKey="saveCategoryForm"/>
         <actionGroup ref="AdminOpenMediaGalleryFromCategoryImageUploaderActionGroup" stepKey="openMediaGalleryFromImageUploader"/>
         <actionGroup ref="ClearFiltersAdminDataGridActionGroup" stepKey="clearFilter"/>
-        <actionGroup ref="AdminMediaGalleryFolderSelectActionGroup" stepKey="selectCatalogFolder">
-            <argument name="name" value="catalog"/>
-        </actionGroup>
+        <conditionalClick selector="{{AdminMediaGalleryFolderSection.folderArrow(catalog)}}"
```

Comment:
Can you please move this conditional click to an action group (i.e.  `AdminMediaGalleryExpandFolder`)